templates/zerver/message_history.html

```diff
@@ -9,6 +9,6 @@ <h3 id="message-history-label">{{ _('Message edit history') }}</h3>
         <div class="controls" id="message-history"></div>
     </div>
     <div class="modal-footer">
-        <button class="btn btn-default" data-dismiss="modal" aria-hidden="true">{{ _("Close") }}</button>
+        <button class="btn btn-default" id="message-history-cancel" data-dismiss="modal" aria-hidden="true">{{ _("Cancel") }}</button>
```

Comment:
Oh yes. Sorry for the carelessness, it just slipped under my nose. I will fix it and let you know.